src/bin/help.ts

```diff
@@ -0,0 +1,230 @@
+import chalk from "chalk";
+
+import { logLine } from "../shared/cli/lines.js";
+
+export function logHelpText(version: string): void {
+	logLine();
+
+	logLine(chalk.cyan(`✨ template-typescript-node-package v${version} ✨`));
+
+	logLine(
+		chalk.cyan(
+			`
+      A quickstart-friendly TypeScript template with comprehensive formatting, 
+      linting, releases, testing, and other great tooling built-in.
+      `,
+		),
+	);
```

Comment:
Thinking on the formatting of this output:

```plaintext

│  ✨ template-typescript-node-package v1.29.30 ✨
│  
      A quickstart-friendly TypeScript template with comprehensive formatting, 
      linting, releases, testing, and other great tooling built-in.
      
│
│  Core options:
│  
      --base: Whether to scaffold the repository with:
        • everything: that comes with the template (recommended)
        • minimum: amounts of tooling, essentially opting out of everything
        • prompt: for which portions to exclude
```

![Screenshot of the CLI output of the previous code block](https://github.com/JoshuaKGoldberg/create-typescript-app/assets/3335181/8303e2be-c7ee-4eb9-9dd7-657a0727adbe)

Note the lack of `|`s on the left. It feels weird to have them only inconsistently. I'm maybe 75% in favor of having them always for consistency & snazzy styling vs. maybe 25% in favor of removing them to keep the help experience simpler. How do you feel?

Reply:
I'm also in two minds about this but leaning towards removing them at least from the help section. Although it is a nice little extra on the styling it feels like it could break more easily on certain terminal shapes and also the help section will be easier to maintain without them.

Reply:
Yeah agreed. Let's get them out of here! 🔪

Reply:
I've removed all the `|`s from the help output lines, just leaving them at the top where they also work with the rest of the tool's UI.